modules/exploits/unix/local/setuid_nmap.rb

```diff
@@ -51,12 +51,11 @@ def initialize(info={})
   end
 
   def check
-    stat = session.fs.file.stat(datastore["Nmap"])
-    if stat and stat.file? and stat.setuid?
-      vprint_good("#{stat.prettymode} #{datastore["Nmap"]}")
-      return CheckCode::Vulnerable
+    if setuid?(datastore['Nmap'])
+      vprint_good("#{datastore['Nmap']} is setuid")
+      CheckCode::Vulnerable
```

Comment:
This needs `return CheckCode::Vulnerable`

Reply:
Oops.